### PR TITLE
completed without pagination

### PR DESCRIPTION
**Pull request by namkatcedrickjumtock**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1 @@
...
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+Google Search Clone
```
Comment (suhfang) [inline README.md:1]: not enough documentation

File: next.config.js [hunk 2]
```diff
@@ -12,4 +12,4 @@ const nextConfig = {
   },
 };
 
-module.exports = nextConfig;
+module.exports = nextConfig;
```
Comment (suhfang) [inline next.config.js:15]: when you see the red sign. be sure to add a new line at the end of every javascript or css file you add to your project or repo.

File: package.json [hunk 3]
```diff
@@ -22,4 +22,4 @@
     "postcss": "^8.4.16",
     "tailwindcss": "^3.1.8"
   }
-}
+}
```
Comment (suhfang) [inline package.json:25]: new line

File: pages/search.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ import MockResponse from "../MockResponse";
 import SearchResults from "../components/SearchResults";
 
 const Search = ({ results }) => {
-  // console.log(results);
+  console.log(results);
```
Comment (suhfang) [inline pages/search.js:8]: no console logs in github

File: styles/Home.module.css [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  .footer {
+    border-color: #222;
+  }
+  .code {
+    background: #111;
+  }
+  .logo img {
+    filter: invert(1);
+  }
+}
```
Comment (suhfang) [inline styles/Home.module.css:129]: preferably use scss instead

File: pages/search.js (hunk 4 above)
Comment (namkatcedrickjumtock) [reply pages/search.js:8]: got it boss

**Pull request merged**